src/stack.py

```diff
@@ -1,34 +1,18 @@
 # -*- coding: utf-8 -*-
-"""Docstring."""
+"""File implements a basic stack data structure."""
 import linked_list
```

Comment:
For a case like this, where you are only using a single object imported from a module, it is more Pythonic to do `from linked_list import LinkedList`. It saves you typing later as you can just type `LinkedList` in the rest of your code instead of `linked_list.LinkedList`